src/types/globals/Window.js

```diff
@@ -34,7 +34,7 @@ declare class Window {
   geckoProfilerAddonInstalled?: () => void;
   isGeckoProfilerAddonInstalled?: boolean;
   InstallTrigger?: {
-    install: MixedObject => {},
+    install: MixedObject => void,
```

Comment:
nit: It looks like this function returns a boolean to indicate that the installation has started successfully: https://searchfox.org/mozilla-central/rev/ff82c973f8ccb0475ec32439e9ec07014b3a681f/dom/webidl/InstallTrigger.webidl#61
I guess it's pretty low priority since we are not using that return value, but it could be still good to type it correctly.

Reply:
Hah, I'll fix it.